src/main/java/com/google/ehub/data/UserDataManager.java

```diff
@@ -0,0 +1,28 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.io.IOException;
+
+public class UserDataManager {
```

Comment:
This class is very similar in behavior to ProfileDatastore.java - both are managing the profile data. You can probably merge the two. 

You can move the method `getUserProfile` into ProfileDatastore.java

Reply:
Ping

Reply:
I think merging the two is a great idea. It will free up any room for confusion.

Reply:
You can delete this entire file now